src/pages/app/searching_project_user/ApplyForm.js

```diff
@@ -5,12 +5,62 @@
  */
 
  //Packages
- import React from "react";
+ import React, {useState} from "react";
+ import { useDispatch } from "react-redux";
+ import { upDataForm } from "../../../store/actions/searching-project-user/project/project";
 
  //Styles
  import "./styles/ApplyFormStyles.css";
 
+
  const ApplyForm = (props) => {
+    //Initialize the states
+    const [Name, setName] = useState("");
+    const [Email, setEmail] = useState("");
+    const [Message, setMessage] = useState("");
+
+    //Dispatch
+    const dispatch = useDispatch();
+
+    const FormState ={
+        Name:Name,
+        Email:Email,
+        Message:Message,
+    }
+   
+    /**
+     * @summary Handle State of Name
+     * @param {string} Name
+     * @return {void}
+     */
+    const onChangeName=(Name) =>{
+        setName(Name);
+    }
+
+    
+    /**
+     * @summary Handle State of Email
+     * @param {string} Email
+     * @return {void}
+     */
+    const onChangeEmail=(Email) =>{
+        setEmail(Email);
+    }
+
+
+    /**
+     * @summary Handle State of Message
+     * @param {string} Message
+     * @return {void}
+     */
+    const onChangeMessage=(Message) =>{
+        setMessage(Message);
```

Comment:
@TrNgTien you seem to misunderstand the MVP flow.
The MVP contains many inputs for each question which are fetched from Long's actions